src/parsers/protobuf/parseMergedMeshes.ts

```diff
@@ -34,7 +35,16 @@ export default function parse(data: ParseData): MergedMeshGroup {
     let triangleOffset = 0;
     nodes.forEach(node => {
       const nodeId = Number(node.properties[0].nodeId);
-      const { color, treeIndex } = node.properties[0];
+      let { treeIndex } = node.properties[0];
+      if (node.properties[0].color == null && !hasWarnedAboutMissingColor) {
+        hasWarnedAboutMissingColor = true;
+        console.warn(
+          '3d-web-parser encountered node with missing color while loading',
+          '(using #ff00ff to highlight objects with missing color).',
+        );
+      }
+      const color = node.properties[0].color != null ? node.properties[0].color : { rgb: 0xff00ff };
```

Comment:
`const { color = { rgb: 0xff00ff } } = node.properties[0];` should work if you're checking for undefined rather than null (but if it's for null then never mind)

Reply:
We are checking for null, unfortunately